pkg/server/servicebroker/apiserver_test.go

```diff
@@ -0,0 +1,35 @@
+package servicebroker
+
+import (
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/require"
+
+	"github.com/operator-framework/operator-lifecycle-manager/pkg/api/client/clientset/versioned/fake"
+)
+
+const (
+	testNamespace = "testns"
+)
+
+func mockALMBroker(ctrl *gomock.Controller) *ALMBroker {
+	return &ALMBroker{
+		client:    fake.NewSimpleClientset(),
+		namespace: testNamespace,
+	}
+}
+
+func TestValidateBrokerAPIVersion(t *testing.T) {
+	ctrl := gomock.NewController(t)
```

Comment:
same here, you don't need ctrl or the defer

Reply:
f2f98478